src/Nordea/Components/Card.elm

```diff
@@ -98,6 +101,7 @@ view attrs children (Card config) =
             baseStyle
             (headerCollapsible (Maybe.values [ config.onClick |> Maybe.map onClick ] ++ attrs)
                 [ config.title |> viewMaybe (\title_ -> Text.bodyTextHeavy |> Text.view [ css [] ] [ Html.text title_ ])
+                , config.htmlTitle |> viewMaybe (\htmlTitle -> Html.div [ css (if config.isOpen then [ marginBottom (rem 1.5) ] else []) ] [ htmlTitle ])
```

Comment:
There is an issue with margin/gap when the card is opened.

<img width="869" alt="image" src="https://github.com/user-attachments/assets/80357fa4-3bdb-4050-8b36-37a1c865afdb">

This is fixed by this conditional marginBottom.
But do we want this here? or should this be controlled by the user? 🤔 
@fredrikloberg maybe knows

Reply:
Hmm, what if you move it to the content container as a marginTop instead, can the conditional be removed then?

Reply:
I remember wanting the title to be of type html when creating this component, but that got shut down. So the result is now that we have quite a lot of custom logic for this card spread around. So I think the margin stuff has to be tested

Reply:
Yes. I actually see now that the current title-stuff is "broken", in that the margin is weird there also.
<img width="1370" alt="image" src="https://github.com/user-attachments/assets/aa7f5609-43e8-41a9-a5d8-460f45c48700">

The issue is the actually the padding

<img width="1506" alt="image" src="https://github.com/user-attachments/assets/df70ce64-d0d7-4b5f-aad3-690174c6f3b4">